lib/utils.js

```diff
@@ -67,17 +67,59 @@ function addArrayToSet(array, set, valueExtractor) {
   return set
 }
 
+function extractLicenseFromLicenseUrl(licenseUrl) {
+  if (!licenseUrl) return
+  for (const licenseUrlOverride of _licenseUrlOverrides) {
+    const licenseUrlMatch = licenseUrlOverride.test.exec(licenseUrl)
+    if (licenseUrlMatch) {
+      if (licenseUrlOverride.license) {
+        return licenseUrlOverride.license
+      } else if (licenseUrlOverride.licenseMatchGroup) {
+        return licenseUrlMatch[licenseUrlOverride.licenseMatchGroup]
+      }
+    }
+  }
+}
+
 function _normalizeVersion(version) {
   if (version == '1') return '1.0.0' // version '1' is not semver valid see https://github.com/clearlydefined/crawler/issues/124
   return semver.valid(version) ? version : null
 }
 
+const _licenseUrlOverrides = [
+  {
+    test: /^\w*https?:\/\/(?:www.)?opensource.org\/licenses\/mit-license(\.php)?$/i,
+    license: 'MIT'
+  },
+  {
+    test: /^\w*https?:\/\/(?:www.)?opensource.org\/licenses\/(.*?)(\.php)?$/i,
+    licenseMatchGroup: 1
```

Comment:
Not sure if it gets handled elsewhere but be careful that the URL isn't case-sensitive but our use of SPDX identifiers *may* be. Same problem exists for the GNU URLs below.

Reply:
ahhh good point. I'll add a test case for this

Reply:
@iamwillbar I brought in the [spdx-license-list](https://www.npmjs.com/package/spdx-license-list) package to validate against to get the right casing all the way through

This is the same package used in the [chive](https://github.com/amzn/tiny-attribution-generator/blob/master/package.json#L20) library